server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -850,8 +850,8 @@ Table buildTable(
             table.addCell(totalStats.getSegments() == null ? null : totalStats.getSegments().getCount());
             table.addCell(primaryStats.getSegments() == null ? null : primaryStats.getSegments().getCount());
 
-            table.addCell(totalStats.getSegments() == null ? null : totalStats.getSegments().getMemory());
-            table.addCell(primaryStats.getSegments() == null ? null : primaryStats.getSegments().getMemory());
+            table.addCell(totalStats.getSegments() == null ? null : totalStats.getSegments().getZeroMemory());
+            table.addCell(primaryStats.getSegments() == null ? null : primaryStats.getSegments().getZeroMemory());
 
```

Comment:
why are we not removing these instead - for backward compatibility?

Reply:
Yes, they're needed for bwc so we just report 0 memory usage.